Project 3/style.css

```diff
@@ -0,0 +1,257 @@
+* {
+  margin: 0;
+  font-family: "Inter", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 50px 130px;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  padding: 0 0 50px 0;
+  align-items: center;
+  width: 100%;
+  margin-top: 80px;
+}
+
+.mobile, .mobile-hamburger {
+  display: none;
```

Comment:
to niestety nie powoduje, że hamburger jest niewidoczny na desktopie. Spróbuj rozkminić czemu :)